TestAssets/DesktopTestProjects/AutoAddDesktopReferencesDuringMigrate/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.CSharp.RuntimeBinder;
+
+namespace AutoAddDesktopReferencesDuringMigrate
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var mscorlib_ref = new List<int>(new int[] { 4, 5, 6 });
```

Comment:
What is up with this naming convetion?

We use camelCasing.

Can you update these here?